bears/go/GoTOMLBear.py

```diff
@@ -0,0 +1,22 @@
+from coalib.bearlib.abstractions.Linter import linter
```

Comment:
TOML is a "minimal configuration file format", so isn't this GoTOMLBear is more appropriate located in `bears/configfiles` rather than in `go` folder?